drivers/oled/oled_driver.c

```diff
@@ -215,10 +225,19 @@ void oled_clear(void) {
 
 static void calc_bounds(uint8_t update_start, uint8_t* cmd_array)
 {
+#if defined(OLED_SH1106)
+  cmd_array[0] = PAM_PAGE_ADDR | (OLED_BLOCK_SIZE * update_start / OLED_DISPLAY_WIDTH);
+  cmd_array[1] = PAM_SETCOLUMN_LSB | (OLED_COLUMN_OFFSET + (OLED_BLOCK_SIZE * update_start % OLED_DISPLAY_WIDTH));
+  cmd_array[2] = PAM_SETCOLUMN_MSB | ((OLED_COLUMN_OFFSET + (OLED_BLOCK_SIZE * update_start % OLED_DISPLAY_WIDTH))>> 4);
```

Comment:
Add a comment describing this a bit:
// Set the starting page and column. Column value requires spiting into two commands.